src/ddmd/dsymbol.d

```diff
@@ -1424,7 +1425,7 @@ public:
                     AliasDeclaration ad = void;
                     // accessing private selective and renamed imports is
                     // deprecated by restricting the symbol visibility
-                    if (s.isImport() || (ad = s.isAliasDeclaration()) !is null && ad._import !is null)
+                    if (cachedSemantics || s.isImport() || (ad = s.isAliasDeclaration()) !is null && ad._import !is null)
```

Comment:
Can't you analyze the cached value in the module it came from? Or what is the problem with the visibility?

Reply:
Well, it's just me being not familiar with DMD's internals so thanks for the tip!